testdata/jsight_0.3/ser-to-openapi/req.japi.http_method.query_0.3/query_basic.openapi.todo

```diff
@@ -0,0 +1,89 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "paths": {
+    "/url": {
+      "get": {
+        "parameters": [
+          {
+            "in": "query",
+            "name": "prim1",
+            "style": "deepObject",
+            "explode": true,
+            "schema": {
+              "type": "integer",
+              "example": 1
+            },
+            "required": true
+          },
+          {
+            "in": "query",
+            "name": "prim2",
+            "style": "deepObject",
+            "explode": true,
+            "schema": {
+              "type": "string",
+              "example": "value"
+            },
+            "required": false
+          },
+          {
+            "in": "query",
+            "name": "arr",
+            "style": "deepObject",
+            "explode": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "integer"
+              }
```

Comment:
Array is not properly converted. You cannot lose the example values (see the convert rules docs https://j-sight.atlassian.net/wiki/spaces/JAPI/pages/402915358#array). It should be:

```
  "items": {
    "anyOf": [
      {
        "type": "integer",
        "example": 2
      },
      {
        "type": "integer",
        "example": 3
      }
    ]
  }
```

Reply:
Fixed.